net/adoptopenjdk/bumblebench/string/StringIndexOf.java

```diff
@@ -0,0 +1,62 @@
+package com.ibm.bumblebench.string;
+
+import com.ibm.bumblebench.core.MicroBench;
+
+import java.util.Random;
+import java.lang.StringBuilder;
+
+/**
+ * 
+ * String.indexOf can be accelerated by vector instructions.
+ * 
+ * This benchmark can generate strings of fixed/variable lengths and invokes
+ * indexOf to measure performance.
+ * 
+ * The reported score is in terms of number of string indexOfs per sec.
+ * 
+ * */
+
+public final class StringIndexOf extends MicroBench {
+
+	private static final int MAX_ITERATIONS_PER_LOOP = option("maxIterations", 10000000);
+	private static final int INDEX_OF_VALUE = option("indexOfValue", 10);
+	private static int STRING_LENGTH = option("stringLength", 0);
+
+	private static String string;
+	private static char[] possibleChars = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l',
```

Comment:
I've seen this block repeated across tests...

Reply:
This block is used in both my open PR, #4, for `StringConversionBench.java` and `StringIndexOf.java`, here, how would you suggest avoiding this duplicated code?

Reply:
Something like a `StringTestData.java`?